annotator/ffa/unknownbinariesanno/filterknownbinaries.go

```diff
@@ -0,0 +1,102 @@
+// Package unknownbinariesanno removes all packages extracted by unknown binaries
+// filters out the known binaries, and records the remaining as a finding.
+package unknownbinariesanno
+
+import (
+	"context"
+	"fmt"
+	"slices"
+
+	"github.com/google/osv-scalibr/annotator"
+	"github.com/google/osv-scalibr/annotator/ffa/unknownbinariesanno/internal/dpkgfilter"
+	"github.com/google/osv-scalibr/annotator/ffa/unknownbinariesanno/internal/filter"
+	"github.com/google/osv-scalibr/extractor"
+	"github.com/google/osv-scalibr/extractor/filesystem/ffa/unknownbinariesextr"
+	"github.com/google/osv-scalibr/inventory"
+	"github.com/google/osv-scalibr/plugin"
+)
+
+// Name of the plugin
+const Name = "ffa/unknownbinaries"
+
+// List of filters to apply to exclude known binaries
+var filters = []filter.Filter{
+	dpkgfilter.DpkgFilter{},
+}
+
+// Annotator further processes the UnknownBinaryExtractor
+type Annotator struct {
+}
+
+// Annotate removes all packages extracted by unknown binaries,
+// filters out the known binaries, and records the remaining as a finding.
```

Comment:
The comment is not accurate anymore - we remove only the known binaries and don't create any findings